src/app/containers/ATIAnalytics/params/topicPage/buildParams.js

```diff
@@ -0,0 +1,29 @@
+import { buildATIPageTrackPath } from '../../atiUrl';
+
+export const buildTopicPageATIParams = (
+  pageData,
+  requestContext,
+  serviceContext,
+) => {
+  const { platform, statsDestination } = requestContext;
+  const { atiAnalyticsAppName, service } = serviceContext;
+
+  return {
+    appName: atiAnalyticsAppName,
+    contentType: 'index-category',
+    pageTitle: pageData.title,
+    statsDestination,
+    platform,
+    service,
+  };
+};
+
+export const buildTopicPageATIUrl = (
+  topicPageData,
+  requestContext,
+  serviceContext,
+) => buildATIPageTrackPath(
+  buildTopicPageATIParams(topicPageData, requestContext, serviceContext),
+);
+
+export default buildTopicPageATIParams;
```

Comment:
Not sure this default export is actually being used - although the equivalent functions of other page types have it 😕

Reply:
Removing it for now - if we need it in the future can just add it back in :)